sdmetrics/reports/single_table/_properties/synthesis.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import pandas as pd
+import plotly.express as px
+
+from sdmetrics.reports.single_table._properties import BaseSingleTableProperty
+from sdmetrics.single_table import NewRowSynthesis
+
+
+class Synthesis(BaseSingleTableProperty):
+    """Synthesis property class for single table.
+
+    This property assesses the novelty of the syntetic data over the real data.
+    The ``NewRowSynthesis`` metric is computed over the real and synthetic table to
+    score the proportion of new rows in the synthetic data.
+    """
+
+    metric = NewRowSynthesis
+
+    def _generate_details(self, real_data, synthetic_data, metadata, progress_bar=None):
+        """Generate the _details dataframe for the synthesis property.
+
+        Args:
+            real_data (pandas.DataFrame):
+                The real data
+            synthetic_data (pandas.DataFrame):
+                The synthetic data
+            metadata (dict):
+                The metadata of the table
+            progress_bar (tqdm.tqdm or None):
+                The progress bar to use. Defaults to tqdm.
+
+        Returns:
+            pandas.DataFrame
+        """
+        name = self.metric.__name__
+        error_message = np.nan
+        if len(synthetic_data) > 10000:
```

Comment:
Why is this here? And should this not log some sort of warning?

It can also be written as one line: `sample_size = len(synthetic_data) if len(synthetic_data) <= 10000 else 10000`

Reply:
@npatki I was also not sure about this.
For the `NewRowSynthesis`, by default, we limit the `sample size` of the synthetic data to 10000 right?
Should we raise a log or a warning if the synthetic data is larger than 10000 rows?

Reply:
I don't think we need this here. Just pass it in as 10000 to the metric. If the data has more than 10000 rows it will subsample, otherwise it will use all the data

Reply:
I did this not to raise this warning (because the user no longer has the possibility to adjust the `sample_size`): https://github.com/sdv-dev/SDMetrics/blob/ddec4f76e3cb866d13b8aded0e319b3e4e96ca3d/sdmetrics/single_table/new_row_synthesis.py#L68

Reply:
Or you can leave it but make it onle line like @fealho suggested

Reply:
Yes alright, done in [4669397](https://github.com/sdv-dev/SDMetrics/pull/398/commits/4669397521b9df075bc08a7d8a4fbbd880177250)